java/clients/src/main/java/sleeper/util/ClientUtils.java

```diff
@@ -101,7 +101,7 @@ private static String splitNonDecimalIntoParts(String str) {
         return String.join(",", parts);
     }
 
-    public static String formatPropertyDescription(String str) {
+    public static String formatDescription(String str) {
```

Comment:
I think this is specific to descriptions of properties. If you pass in the property we can avoid saying that in the method name, but we should probably make it clear what it's for.